api/pom.xml

```diff
@@ -16,7 +16,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>referencemetadata</artifactId>
-		<version>2.9.0-SNAPSHOT</version>
+		<version>2.10.0-SNAPSHOT</version>
```

Comment:
Is this change part of the ticket?

Reply:
oop s , that was a mistake , thats unnecesary. let me clean it out